vnext/Microsoft.ReactNative/RedBox.cpp

```diff
@@ -142,7 +142,20 @@ struct RedBox : public std::enable_shared_from_this<RedBox> {
   }
 
  private:
-  void UpdateErorrMessageUI() noexcept {
+  static bool IsMetroBundlerError(const std::string &message, const std::string &type) {
+    // This string must be kept in sync with the one in formatBundlingError in
+    // node_modules\metro\src\lib\formatBundlingError.js
+    if (message.find_first_of("Metro Bundler has encountered an error") != message.npos) {
+      return true;
+    }
+    return false;
+  }
+
+#define METRO_TROUBLESHOOTING_URL "http://aka.ms/RNWTroubleshootMetro"
```

Comment:
Should this be part of react-native upstream rather than react-native-windows?  Or even part of metro?

Reply:
Possibly although I imagine some of the problems we run into are specific to RN and to RNW. We can always re-point the link to something else. For the time being since we're iterating quickly on the content, I'll keep this on our site until it's a little more stable, at which point we can migrate it to either RN or Metro's.